grouper/fe/handlers.py

```diff
@@ -732,6 +743,16 @@ def get(self):
             .filter(Group.enabled == True)
             .order_by(Group.groupname)
         )
+        if audited_only:
+            audited_permissions = (
+                self.session.query(Permission)
+                .filter(Permission._audited == True).all()
+            )
+            groups = groups.filter(
```

Comment:
I see you updated this, but why are you doing the queries manually instead of using the graph which contains all of this data calculated? Granted, the graph means you're 10s delayed or whatever the reload interval is, but that's probably better than doing this query manually?